homeassistant/components/goe_charger/__init__.py

```diff
@@ -0,0 +1,91 @@
+"""The go-e Charger integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from homeassistant.core import HomeAssistant
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+from homeassistant.helpers import device_registry
+
+from .const import DOMAIN
+from .common import GoeChargerHub
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[str] = ["binary_sensor", "number", "select", "sensor"]
```

Comment:
You should only include one platform in the initial pr
https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible

Reply:
But the hardware has so many states and settings to offer. I don't want to delete already implemented functionality. Is this really needed?

Reply:
I do not think it is required, but it will take time before you get a review.
> Pull requests containing large code dumps will not be a priority for review and may be closed

Reply:
I would recommend excluding some of the code from this pull request and then you can open a new pull request when this is merged.